packages/template-retail-react-app/app/components/links-list/index.jsx

```diff
@@ -45,11 +45,27 @@ const LinksList = ({
                 ))}
 
             {links && (
-                <List spacing={5} {...styles.list}>
+                <>
                     {variant === 'horizontal' ? (
-                        <HStack>
+                        <>
+                            <List {...styles.list}>
+                                {links.map((link, i) => (
+                                    <ListItem key={i} {...styles.listItem} sx={styles.listItemSx}>
+                                        <Link
+                                            to={link.href}
+                                            onClick={onLinkClick}
+                                            {...(link.styles ? link.styles : {})}
+                                        >
+                                            {link.text}
+                                        </Link>
+                                    </ListItem>
+                                ))}
+                            </List>
+                        </>
```

Comment:
Minor: in this case, we can safely delete this empty component, right?

Reply:
Gotcha. It is removed as suggest.